lib/iris/tests/unit/coords/__init__.py

```diff
@@ -27,11 +27,12 @@
 
 import dask.array as da
 import numpy as np
+import numpy.ma as ma
 
 from iris._lazy_data import is_lazy_data
 
 
-def setup_test_arrays(self, shape):
```

Comment:
It's questionable whether we need to bother switching 'masked' on + off at all.
We could just *always* do the masked stuff ...
On my desktop, tested with `python -m unittest discover -v iris.tests.unit.coords` 
 * with 'masked' switch takes 0.560 - 0.570
 * with switching disabled (mask==True), takes 0.635-0.650 secs

So the cost is jolly small !!

Reply:
Yup get your point, but I'm just going to leave it as is, if you don't mind :stuck_out_tongue_winking_eye: